wapitiCore/net/crawler.py

```diff
@@ -372,13 +376,28 @@ async def _async_try_login_basic_digest_ntlm(self, auth_url: str) -> Tuple[bool,
 
         if page.status in (401, 404):
             return False, {}
-        return True, {}
+        return True, {}, []
+
+    @staticmethod
+    def _extract_disconnect_urls(page: Page) -> List[str]:
+        """
+        Extract all the disconnect urls on the given page and returns them.
+        """
+        disconnect_urls = []
+        for link_tag in page.soup.find_all('a'):
```

Comment:
You should use `page.links` here instead as it will directly provide absolute urls. Also you should maybe filter using `crawler.is_in_scope` to reduce false positives.